crates/autopilot/src/domain/competition/mod.rs

```diff
@@ -0,0 +1,66 @@
+use {
+    super::{auction, eth},
+    crate::domain,
+    number::nonzero::U256 as NonZeroU256,
+    std::collections::HashMap,
+};
+
+type SolutionId = u64;
+
+pub struct Solution {
+    id: SolutionId,
+    account: eth::Address,
```

Comment:
`account` is not very specific. Wouldn't hurt to call it `solver`.

Reply:
Covered in https://github.com/cowprotocol/services/pull/2746